devicetypes/smartthings/aeon-home-energy-meter.src/aeon-home-energy-meter.groovy

```diff
@@ -157,13 +159,12 @@ def configure() {
 	log.debug "configure()..."
 	if (isAeotecHomeEnergyMeter())
 		delayBetween([
-				encap(zwave.configurationV1.configurationSet(parameterNumber: 255, size: 4, scaledConfigurationValue: 1)), // Reset the device to the default settings
```

Comment:
I think we'd want to keep this, since it guarantees we get consistent settings on configure.

Unless you can confirm whether or not settings are preserved on removing and re-adding the device to a network.

Reply:
All default parameter settings will revert back to their default values upon removing and re-adding the device to a network, there are no persistent parameter settings.

Reply:
great, then this is fine.